### PR TITLE
Feature/notifications field filtering

### PR DESCRIPTION
**Pull request by khatton-ft**
Update filter so that notificationDate follows the same rules as lastModified.

### CODE REVIEW

File: api-policy-component-service/pom.xml [hunk 1]
```diff
@@ -53,7 +53,7 @@
         <maven-shade-plugin.version>.</maven-shade-plugin.version>
         <maven-resources-plugin.version>2.6</maven-resources-plugin.version>
         <maven-jar-plugin.version>2.3.2</maven-jar-plugin.version>
-        <target-jdk>1.7</target-jdk>
+        <target-jdk>1.8</target-jdk>
```
Comment (peteclark-ft) [inline api-policy-component-service/pom.xml]: Is this reflected in the `Dockerfile`?

**Comment by peteclark-ft** [comment]
Orrr maybe we should have a [graphQL](http://graphql.org/) server

File: api-policy-component-service/pom.xml (hunk 1 above)
Comment (khatton-ft) [inline api-policy-component-service/pom.xml]: Yes, the base Docker layer includes JDK8 anyway.

**Comment by khatton-ft** [comment]
I agree that it feels like it should be possible to do what we want with "buy" rather than "build". But Json-view is GPL and I don't see Json Patch really doing what we need. I believe Next use GraphQL - maybe you could create a 10% task to investigate whether it would be useful for us?

**Pull request merged**